scripts/fu_storageutils.lua

```diff
@@ -6,23 +12,21 @@ function fu_storeItems(items, avoidSlots, spawnLeftovers)
 
 		for i = 0, world.containerSize(entity.id()) do -- iterate all output slots
 			local stack = world.containerItemAt(entity.id(), i) -- get the stack on i
-			if stack then -- not empty
-				if stack.name == something then -- its "something"
-					table.insert(slots,i) -- possible drop slot
-				end
+			if stack and (stack.name == something) then -- not empty and its "something"
+			  table.insert(slots,i) -- possible drop slot
```

Comment:
While what you write is true in and of itself, there's an `else` block to consider, and doing the rewriting which you propose changes the condition required for execution of that block from `not condition1` to `not condition1 or not condition2`. The implicit ‘do nothing’ in the case of `stack and stack.name ~= something` _is intended_.

So, for that change, still NAK.